wavepool/models.py

```diff
@@ -32,7 +32,17 @@ def teaser(self):
 
     @property
     def source_divesite_name(self):
-        return 'Industry Dive'
+        """ Return the real divesite source name for the newspost's source
+        """
+        source_dive = 'Industry Dive'
+        for dive_source in DIVESITE_SOURCE_NAMES:
+            url_parts = self.source.split('/')
+            dive_domain = url_parts[2]
+            domain_parts = dive_domain.split('.')
+            dive_domain = domain_parts[1]
+            if dive_domain == dive_source:
+                return DIVESITE_SOURCE_NAMES[dive_domain]
+        return source_dive
```

Comment:
I would suggest that it may make more sense to extract the domain name using regex first, and then use that as a key to look up the divesite name. It should make the code a little more concise.

Here is a [SO post](https://stackoverflow.com/questions/55862019/extract-domain-name-from-url-using-pythons-re-regex) trying to do a similar thing.